cmd/glasskube/cmd/bootstrap.go

```diff
@@ -31,6 +33,30 @@ var bootstrapCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg, _ := cliutils.RequireConfig(config.Kubeconfig)
 		client := bootstrap.NewBootstrapClient(cfg)
+
+		installedVersion, err := clientutils.GetPackageOperatorVersion(cmd.Context())
+		if err != nil {
+			IsBootstrapped, err := bootstrap.IsBootstrapped(cmd.Context(), cfg)
+			if err != nil && !IsBootstrapped {
+				fmt.Printf("error : %v\n", err)
+				return
+			}
+			return
+		}
+
+		var desiredVersion string
+		if bootstrapCmdOptions.url == "" {
+			desiredVersion = config.Version
+		} else {
+			desiredVersion = bootstrapCmdOptions.url
+		}
+		if !semver.IsUpgradable(installedVersion, desiredVersion) {
```

Comment:
in this `if` please add the condition `installedVersion != ""`, because we don't need to check this if the package operator is not installed yet.